via/static/scripts/video_player/index.tsx

```diff
@@ -19,6 +19,27 @@ export function init() {
     video_id: videoId,
   } = readConfig();
 
+  // When content is displayed in an iframe, notify top-level of title and
+  // location. This also acts to tell the top-level frame that we finished
+  // loading. See proxy.html.jinja2.
+  if (window !== window.top) {
+    window.parent.postMessage(
+      {
+        type: 'metadatachange',
+        metadata: {
+          // TODO: Once https://github.com/hypothesis/via/pull/1015 lands,
+          // get the URL from `link[rel=canonical]` or pass the video URL as
+          // config.
+          location: `https://www.youtube.com/watch?v=${encodeURIComponent(
+            videoId
+          )}`,
+          title: document.title,
```

Comment:
Confirm: expected `title` is `'Via: Hypothesis Video Player'`?

Reply:
Currently, yes. "Hypothesis Video Player" comes from the video player and the "Via:" prefix is always added by Via to distinguish a web page visited directly from a web page proxied through Via.